### DISCUSSION TITLE
[Feature Request] - Adjust Nagging Notification Timing Per Task

### DISCUSSION THREAD

**Discussion by mschwerz** [donetick/donetick#72] [Ideas]
It would be nice to be able to set how often a Nagging notification is sent per each task

**Comment by raisinbear** [comment]
Came here to ask for this! :) I see by default, 24, 48, and 72 hours are passed to the planner. Would be great if that could be customized in some way as for some reminders, it can be crucial to be nagged hard every hour or so if you miss the notification or are preoccupied.

**Comment by meauxt** [comment]
Yeah, this is also related to https://github.com/donetick/donetick/discussions/193  ?
Started looking into this last year and will pick up that work soon. Here is what the state of things is at this point and what I am thinking.
You would have a default notification template that you can use, and you can create your own. This notification template can be assigned to tasks and can be reused  
![image](https://github.com/user-attachments/assets/3d660c1f-236f-4832-9583-5f2c2dddfb89)
This might change; I still need to think more about it. don't want to complicate things, but also want to give flexibility

**Reply by raisinbear** [reply]
Oh yeah right, sorry stumbled upon this one first.
But your suggestion looks awesome and even more flexible than I was thinking 👍

**Reply by dani315p00n** [reply]
> Yeah, this is also related to #193 ? Started looking into this last year and will pick up that work soon. Here is what the state of things is at this point and what I am thinking. You would have a default notification template that you can use, and you can create your own. This notification template can be assigned to tasks and can be reused ![image](https://private-user-images.githubusercontent.com/14058741/451165159-3d660c1f-236f-4832-9583-5f2c2dddfb89.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMyMDgwNDMsIm5iZiI6MTc1MzIwNzc0MywicGF0aCI6Ii8xNDA1ODc0MS80NTExNjUxNTktM2Q2NjBjMWYtMjM2Zi00ODMyLTk1ODMtNWYyYzJkZGRmYjg5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzIyVDE4MDkwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjZTg2N2U3MTEyNzA0N2U1YjZlNjMxNjk1NmY0MTNlYTkyODc5MWEzZjJkZjBiNzllMDdhOTg2Yjk5YTIyODgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.BqylLWQhqlr1z4aWQx6Aq5VQv7C6PKVq-D6DbjBvXA0) This might change; I still need to think more about it. don't want to complicate things, but also want to give flexibility

I like the current notification options as it is right now which matches your screenshot! Only change I'd like to request is a time interval in perpetuity for After Due date. Right now it seems like I can add up to 4 After Due notifications so I'm using 1, 2, 4, & 7 but I'd really like a daily reminder until the task is completed.
